warehouse/query-core/src/test/java/datawave/query/jexl/visitors/LiteralNodeSubsetVisitorTest.java

```diff
@@ -0,0 +1,180 @@
+        
+        test(queryString, literals);
+    }
+    
+    @Test
+    public void testMultipleLiteralsAllMatchAndRegex() throws ParseException {
+        String queryString = "FOO == '1234' && BAR == 'abcd' || BAZ =~ 'abcd.*\\d'";
+        Multimap<String,String> literals = ArrayListMultimap.create();
+        
+        literals.put("FOO", "1234");
+        literals.put("BAR", "abcd");
+        test(queryString, literals);
+    }
+    
+    @Test
+    public void testMultipleLiteralsMixedMatchAndRegex() throws ParseException {
+        String queryString = "FOO == '1234' && FLIM == 'flam' || BAZ =~ 'abcd.*\\d'";
+        Multimap<String,String> literals = ArrayListMultimap.create();
+        
+        literals.put("FOO", "1234");
+        test(queryString, literals);
+    }
+    
+    @Test
+    public void testMultipleLiteralsNoMatchAndRegex() throws ParseException {
+        String queryString = "FLIM == 'flam' && PARA == 'diddle' || CHEESE =~ 'ka.*\\d'";
+        Multimap<String,String> literals = ArrayListMultimap.create();
+        
+        test(queryString, literals);
+    }
+    
+    @Test
+    public void testExcludeRegex1() throws ParseException {
+        String query = "filter:excludeRegex(FOO,'ba.*')";
+        Multimap<String,String> literals = ArrayListMultimap.create();
+        
+        test(query, literals);
+    }
+    
+    @Test
+    public void testExcludeRegex2() throws ParseException {
+        String query = "FOO == 'bar' && filter:excludeRegex(FOO,'ba.*')";
+        Multimap<String,String> literals = ArrayListMultimap.create();
+        literals.put("FOO", "bar");
+        
+        test(query, literals);
+    }
+    
+    @Test
+    public void testExcludeRegex3() throws ParseException {
+        String query = "FOO == 'bar' || filter:excludeRegex(FOO,'ba.*')";
+        Multimap<String,String> literals = ArrayListMultimap.create();
+        literals.put("FOO", "bar");
+        
+        test(query, literals);
+    }
+    
+    @Test
+    public void testIncludeRegex1() throws ParseException {
+        String query = "filter:includeRegex(FOO,'ba.*')";
+        Multimap<String,String> literals = ArrayListMultimap.create();
+        
+        test(query, literals);
+    }
+    
+    @Test
+    public void testIncludeRegex2() throws ParseException {
+        String query = "FOO == 'bar' && filter:includeRegex(FOO,'ba.*')";
+        Multimap<String,String> literals = ArrayListMultimap.create();
+        literals.put("FOO", "bar");
+        
+        test(query, literals);
+    }
+    
+    @Test
+    public void testIncludeRegex3() throws ParseException {
+        String query = "FOO == 'bar' || filter:includeRegex(FOO,'ba.*')";
+        Multimap<String,String> literals = ArrayListMultimap.create();
+        literals.put("FOO", "bar");
+        
+        test(query, literals);
+    }
+    
+    @Test
+    public void testFunctionInsideQueryPropertyMarker() throws ParseException {
```

Comment:
Rename this method to reflect the query, i.e., `testDelayedEqNode` or `testEqNodeInsideQueryPropertyMarker`

Reply:
Done and done.